src/main/java/seedu/application/model/application/UpcomingInterviewPredicate.java

```diff
@@ -0,0 +1,35 @@
+package seedu.application.model.application;
+
+import java.time.LocalDateTime;
+import java.util.Optional;
+import java.util.function.Predicate;
+
+/**
+ * Tests that an {@code Application}'s {@code Interview} is within 1 week from now.
+ */
+public class UpcomingInterviewPredicate implements Predicate<Application> {
+
+    /**
+     * Returns true if interview date and time is within one week from current date-time,
+     * using system clock in the default time-zone.
+     */
+    public static boolean isWithinOneWeekFromNow(LocalDateTime interviewDateTime) {
```

Comment:
Perhaps this should be made private instead?
```suggestion
    private static boolean isWithinOneWeekFromNow(LocalDateTime interviewDateTime) {
```